server/README.md

```diff
@@ -77,6 +77,10 @@ scratchy-server
 <scratchy/docker>$ docker-compose up -d
 ```
 
+## how to run unitary test
```

Comment:
This is integration test.
Please, could you rename this title in the documentation.